### ISSUE TITLE
Table parameter not working when passed from one concept to another

### ISSUE THREAD

**Issue by singhverse** [getgauge/gauge#1485]
# Expected behavior
When a concept specifies another concept as a step with table parameter, the table must be passed as an instance of **getgauge.python.Table** class.

### Actual behavior
The table parameter is received as **str** with **empty** value.
Gauge throws an `AttributeError` upon accessing **getgauge.python.Table** methods on it.

### Steps to reproduce
1. Create a **concept** that accepts a table parameter and uses it in one of the steps.
2. Create another concept (same or different **.cpt** file) that uses concept from step 1 as a **step** and passes an `inline table parameter` to it.
3. Use concept from step 2 as a **step** in your **scenario** definition
4. Run the **scenario**

### Gauge version
```
Gauge version: 1.0.6

Plugins
-------
html-report (4.0.8)
python (0.3.5)
screenshot (0.0.1)
```

**Comment by singhverse** [comment]
Sample code snippets:

**example.spec** file
```
# Specification Heading

## Vowel counts in multiple word

* Almost all words have vowels
```

**concept1.cpt** file
```
# Almost all words have vowels
* Another abstraction for all words have vowels
     |Word  |Vowel Count|
     |------|-----------|
     |Gauge |3          |
     |Mingle|2          |
     |Snap  |1          |
     |GoCD  |1          |
     |Rhythm|0          |

# Another abstraction for all words have vowels <vowels_table>
* assert all words have vowels <vowels_table>
```

**step_impl.py** file
```
@step("assert all words have vowels <table>")
def assert_words_vowel_count(table):
    actual = [str(number_of_vowels(word)) for word in table.get_column_values_with_name("Word")]
    expected = [str(count) for count in table.get_column_values_with_name("Vowel Count")]
    assert expected == actual
```

**Comment by singhverse** [comment]
Hi @shubhamsc Is there any estimate as to when the team will be able to take this up? I'm not fluent with Go otherwise would be happy to help.

Also, meanwhile, is there any workaround that you can suggest? Maybe a previous version where this issue is not present.

**Comment by Apoorva-GA** [comment]
Fix available in commit 39e4399c97505b275d21a10b9711fc731c40edf7

**Comment by Debashis9012** [comment]
This issue has been fixed and verified.
Tested version: 
`Gauge version: 1.0.7
Commit Hash: 63760ca7`
